src/templates/en/2019/chapter.html

```diff
@@ -1,12 +1,12 @@
-{# IMPORTANT!
+<!--  IMPORTANT!
```

Comment:
Not sure we want this. As a Jinja comment, it is excluded from rendering. As an HTML comment, won't it be included in the page source and visible to users?

Reply:
You're right, but it results in a parsing error. I'll try and resolve it another way.

Reply:
I've embedded the jinja comment inside the html comment, so that jinja will strip it out, but we will still send `<!---->` down as a part of the document.

The `<!-- prettier-ignore -->` functionality is also a code comment, so would send more characters down with the document, for the same net effect.

Do you think this is an acceptable compromise, or shall I keep looking for a different solution?

Reply:
I think this is a reasonable tradeoff. My only concern is that the comment appears as the first line of the document:

```html
<!---->

<!doctype html>
<html lang="en-US">
  <head>
    <meta charset="UTF-8">
```

I think that would screw up the `doctype` declaration.

Reply:
I moved the `{% extends %}` to the top of the file (above the comment), and now jinja is not generating the empty html comment at all. It's a bit bizarre, but I think that solves the issue?